.github/workflows/xk6.yml

```diff
@@ -17,20 +17,26 @@ jobs:
       matrix:
         go: [stable, tip]
         platform: [ubuntu-latest, windows-2019, macos-latest]
-    runs-on: ubuntu-latest
```

Comment:
Looks like we weren't using `matrix.platform`, so actually all the jobs were being executed on the same platform 😅 

I realized it, because when I tried to use the precompiled Go tip, it complained because the binary wasn't in the expected platform (e.g. a binary built for Windows, trying to be executed on Ubuntu, in this case haha)

Reply:
Good catch!